scripts/utilities/004_recompute_clearing_state_of_release.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2021. Part of the SW360 Portal Project.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# This is a manual database migration script. It is assumed that a
+# dedicated framework for automatic migration will be written in the
+# future. When that happens, this script should be refactored to conform
+# to the framework's prerequisites to be run by the framework. For
+# example, server address and db name should be parameterized, the code
+# reorganized into a single class or function, etc.
+#
+# This script gets all release which has 1 or more attachment.
+# iterates over the attachment and recomputes clearing state of release.
+# -----------------------------------------------------------------------------
+
+import time
+import couchdb
+import json
+from webbrowser import get
+
+# ---------------------------------------
+# constants
+# ---------------------------------------
+
+DRY_RUN = True
+
+COUCHSERVER = "http://admin:root@localhost:5984/"
+DBNAME = 'sw360db'
+
+couch = couchdb.Server(COUCHSERVER)
+db = couch[DBNAME]
+
+# ----------------------------------------
+# queries
+# ----------------------------------------
+
+# get all releases with attachment
+all_releases_with_attachments_query = {"selector": {"type": {"$eq": "release"}},"limit": 200000}
```

Comment:
There seems to be no check for attachment in this query. Please add it or rename the variable.

Reply:
Done